File: DSA/SuryanshMishra_590012069/Day9/day9question2.py
def main_vowel_consonant():
    s = input("Enter a string: ")
    vowels = 0
    consonants = 0
    
    for c in s:
        if c.isalpha():
            lower_char = c.lower()
            if lower_char in 'aeiou':
                vowels += 1
            else:
                consonants += 1
    
    print(f"Vowels: {vowels}")
    print(f"Consonants: {consonants}")

File: DSA/Rahul_Mankani_590014725/Day9_Question2.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() 
{
    char str[100];
    int vowels = 0;
    int consonants = 0;
    printf("Enter a string: ");
    fgets(str, sizeof(str), stdin);
    if (str[strlen(str) - 1] == '\n') 
    {
        str[strlen(str) - 1] = '\0';
    }
    for (int i = 0; str[i] != '\0'; i++) 
    {
        char ch = tolower(str[i]);
        if (isalpha(ch)) 
        {
            if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') 
            {
                vowels++;
            } 
            else 
            {
                consonants++;
            }
        }
    }
    printf("Vowels: %d\n", vowels);
    printf("Consonants: %d\n", consonants);
    return 0;
}
